app/assets/stylesheets/osem-schedule.css.scss

```diff
@@ -214,6 +214,15 @@ td.no-padding{
   display: inline-block;
 }
 
+.title{
+  font-size: 23px;
+}
+
+.subtitle{
```

Comment:
Because the `h3` has margin at the top and the `panel` padding too, so it is too much margin. They were `h3` and `small` before but it looks horrible:

![image](https://cloud.githubusercontent.com/assets/16052290/16500365/8bcc3826-3f04-11e6-9336-74f8f564c771.png)